docs/config.md

```diff
@@ -230,6 +230,9 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Skips the generation of summary reports and tables
 [ skip_summary_reports: <bool> | default: false]
 
+# Skips the generation of detailed reports and tables
+[ skip_detailed_reports: <bool> | default: false]
```

Comment:
Is there anything else in our docs that need to change to mention that we now automatically create the reports, and that it utilizes num_configs_per_model?

Reply:
report.md?

Reply:
I've fixed the `report.md` documentation